.github/workflows/odh-release.yaml

```diff
@@ -0,0 +1,48 @@
+# This workflow will compile e2e tests and release them
+
+name: ODH Release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Tag to be used for release, i.e.: v0.0.1-odh-1'
+        required: true
+
```

Comment:
In both kueue and kuberay we also run the release workflow when a tag is added - do you think we should do this here too?
[Example](https://github.com/opendatahub-io/kueue/blob/dev/.github/workflows/odh-release.yml#L10-L12)

Reply:
you are right, I think we should, thanks Fiona!